apps/randompokedex/random_pokedex.star

```diff
@@ -0,0 +1,183 @@
+"""
+Applet: Random Pokedex
+Summary: Random Pokedex entry
+Description: Display a random Pokemon along with its typing and optional shiny version.
+Author: Kerry Bassett
+"""
+
+load("cache.star", "cache")
+load("encoding/base64.star", "base64")
+load("encoding/json.star", "json")
+load("http.star", "http")
+load("random.star", "random")
+load("render.star", "render")
+load("schema.star", "schema")
+
+NUM_POKEMON = 1010
+POKEAPI_URL = "https://pokeapi.co/api/v2/pokemon/{}"
+SPRITE_URL = "https://img.pokemondb.net/sprites/home/normal/"
+SHINY_SPRITE_URL = "https://img.pokemondb.net/sprites/home/shiny/"
+CACHE_TTL_SECONDS = 3600 * 24 * 7  # 7 days in seconds.
+
+TYPE_COLORS = {
+    "normal": "#AA9",
+    "fire": "#F40",
+    "water": "#39F",
+    "electric": "#FC3",
+    "grass": "#7C5",
+    "ice": "#6CF",
+    "fighting": "#B54",
+    "poison": "#A59",
+    "ground": "#DB5",
+    "flying": "#89F",
+    "psychic": "#F59",
+    "bug": "#AB2",
+    "rock": "#BA6",
+    "ghost": "#66B",
+    "dragon": "#76E",
+    "dark": "#754",
+    "steel": "#AAB",
+    "fairy": "#E9E",
+    "": "#000",
+}
+
+def main(config):
+    id_ = random.number(1, NUM_POKEMON)
```

Comment:
```suggestion
    random.seed(time.now().unix // 15)
    id_ = random.number(1, NUM_POKEMON)
```

Reply:
This should make the app just a little bit "less random", which helps us a lot when it comes to caching the rendered graphics.

Reply:
Yeah this seems to return the same pokemon number for 15 seconds before it gets a new random number.